app/models/olympian.rb

```diff
@@ -11,4 +11,19 @@ class Olympian < ApplicationRecord
   validates :height, presence: true
   validates :weight, presence: true
   validates :team, presence: true
+
```

Comment:
You might considered making a scope for the `OlympianEvent` model, something like `scope :medalled_efforts, -> { where.not(medal: 'NA') }`